### DISCUSSION TITLE
Is Forth syntax highlighting  available ?

### DISCUSSION THREAD

**Discussion by kbmaniac** [pulsar-edit/.github#295] [Pulsar Support] [answered]
The title says it all .... a long while ago I used atom and I think it supported Forth, does Pulsar ?

**Comment by savetheclocktower** [comment, marked as answer]
I haven't used it personally, but there is a [language-forth](https://web.pulsar-edit.dev/packages/language-forth) community package.

**Reply by kbmaniac** [reply]
Thank you ... I took the plunge, installed pulsar and the forth highlighting ... Think I have found my new home 👍
